app/_hub/moesif/kong-plugin-moesif/_metadata.yml

```diff
@@ -65,6 +67,8 @@ kong_version_compatibility: # required
     #incompatible:
   enterprise_edition: # optional
     compatible:
+      - 3.2.x
```

Comment:
I don't think the plugin smoke tests were failing because you added 3.3.x. They're flaky occasionally and seems like they just timed out. So you should be able to list 3.3.x support now.

Reply:
Ok will try to readd